ci-operator/config/cri-o/test/cri-o-test-main.yaml

```
@@ -0,0 +1,40 @@
+base_images:
+  coreos-centos-stream8:
+    name: stream8
+    namespace: coreos
+    tag: "8"
```

```suggestion
    tag: "8"
  dev-scripts:
    name: test
    namespace: ocp-kni
    tag: dev-scripts
```

The ``baremetalds-packet-setup`` step requires this image to be "imported", otherwise it will not work